packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -74,6 +76,19 @@ const COLUMN_STATE = {
   ),
   name: _('remoteState'),
 }
+const COLUMN_DISK = {
+  itemRenderer: (remote, { formatMessage }) => (
+    <span>
+      {remote.info &&
+        remote.info.used !== undefined &&
+        remote.info.available !== undefined &&
+        `${formatSize(remote.info.used)} / ${formatSize(
+          remote.info.available
+        )}`}
```

Comment:
Is `remote.info.available` the total space of the remote or just the free space?

Reply:
It's the free space available, not total.